src/main.tsx

```diff
@@ -5,8 +5,14 @@ import './css/style.css'
 
 import App from './App.tsx'
 
-window.Tido = function Tido(config = {}) {
-    createRoot(document.getElementById('app')!).render(<App customConfig={config} />)
+declare global {
```

Comment:
@paulpestov I merged the PR, but I have still one question. Which is the idea behind the lines 8-12 in this file ?

Reply:
Just to satisfy Typescript